package.json

```diff
@@ -43,7 +43,7 @@
     "fs": false
   },
   "engines": {
-    "node": ">= 14"
+    "node": ">= 17"
```

Comment:
@jamesmoschou this caused a regression on our end as well. just having it in the tree of a node v16.x project breaks during dependency installation (yarn, in our case), even if we don't directly use this package.